.travis.yml

```diff
@@ -26,38 +26,37 @@ cache:
     - $HOME/.composer/cache/files
 
 env:
-  matrix: SYMFONY_VERSION=4.0.*
+  matrix: SYMFONY_VERSION="4.2.*"
   global:
-    - SYMFONY_DEPRECATIONS_HELPER=0
+    - SYMFONY_DEPRECATIONS_HELPER="weak"
     - SYMFONY_PHPUNIT_DIR=.phpunit SYMFONY_PHPUNIT_REMOVE="symfony/yaml"
-    - KERNEL_CLASS=Symfony\Cmf\Bundle\SonataPhpcrAdminIntegrationBundle\Tests\Fixtures\App\Kernel
-    - SYMFONY_PHPUNIT_VERSION=5.7
-    - TEST_INSTALLATION=false
+    - KERNEL_CLASS="Symfony\Cmf\Bundle\SonataPhpcrAdminIntegrationBundle\Tests\Fixtures\App\Kernel"
```

Comment:
please move this into phpunit.xml.dist so the tests run locally. you can remove KERNEL_DIR in phpunit.xml.dist,  because that is no longer used.